src/components/audit/row.vue

```diff
@@ -106,6 +106,7 @@ export default {
       const species = acteeSpeciesByCategory[this.category];
       if (species == null) return null;
       const { actee } = this.audit;
+      if (actee == null) return null;
```

Comment:
Unlike other user actions, `user.session.create` isn't associated with an actee. The user who logged in is the actor. I could also/instead make the user the actee, but that seemed potentially confusing.

Reply:
i think we usually in this case set the actor as both the actor and actee. then we don't have the null.